Document AI Obs End-user Feedback Feature (#37342)
DataDog/documentation: content/en/llm_observability/evaluations/external_evaluations.md

</div>

> <div class="alert alert-info">For thumbs-up or thumbs-down ratings, accepted changes, free-text comments, and other signals submitted by users or agent submitters, see <a href="/llm_observability/evaluations/end_user_feedback/">End-User Feedback</a>.</div>

Author: ```suggestion
<div class="alert alert-info">For feedback submitted by your users such as thumbs-up or thumbs-down ratings, accepted changes, free-text comments, and other signals, see <a href="/llm_observability/evaluations/end_user_feedback/">End-User Feedback</a>.</div>
```

        {
          "id": "cdfc4fc7-e2f6-4149-9c35-edc4bbf7b525",
          "join_on": {
            "span": {
              "span_id": "20245611112024561111",

>               "trace_id": "13932955089405749200"

Author: ```suggestion
            "tag": {
              "key": "msg_id",
              "value": "1123132"
```
